apps/system/js/app_window.js

```diff
@@ -2431,5 +2448,63 @@
       this.statusbar.handleStatusbarTouch(evt, barHeight);
     }
   };
+
+
+  /**
+   * Return a promise that resolves to an icon URL.
+   *
+   * @param {number?} iconSize
+   * @returns {Promise}
+   */
+  AppWindow.prototype.getSiteIconUrl =
+    function ac_getSiteIconUrl(iconSize = ICON_SIZE) {
```

Comment:
Actually, the iconSize is passed through to IconsHelper.getIcon which multiplies by window.devicePixelRatio, so it is already taken into account